File: Exercies/06_for/e03_displayMiddle.py
num = int(input("Give odd number between <5,11>: "))
if 5 <= num <= 11 and num % 2 == 1:
    stars = "*" * ((num - 1) // 2)
    print(f"{stars}I{stars}\n"*num,end="")
else:
    print("not a good number")

File: Answers/06_for/e03_displayMiddle.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Give odd number between <5,11>: ");
        if (int.TryParse(Console.ReadLine(), out int num) && num >= 5 && num <= 11 && num % 2 == 1)
        {
            string stars = new string('*', (num - 1) / 2);
            for (int i = 0; i < num; i++)
            {
                Console.WriteLine($"{stars}I{stars}");
            }
        }
        else
        {
            Console.WriteLine("not a good number");
        }
    }
}
